cmd/neofs-node/config/control/config.go

```diff
@@ -0,0 +1,47 @@
+package controlconfig
+
+import (
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+)
+
+type (
+	GRPCConfig struct {
+		cfg *config.Config
+	}
+)
+
+const (
+	subsection     = "control"
+	grpcSubsection = "grpc"
+
+	GRPCEndpointDefault = ""
+)
+
+// AuthorizedKeysString returns string array of "authorized_keys" config
+// parameter from "control" section.
+//
+// Returns empty list if not set.
+func AuthorizedKeysString(c *config.Config) []string {
+	return config.StringSliceSafe(c.Sub(subsection), "authorized_keys")
+}
+
+// GRPC returns structure that provides access to "grpc" subsection of
+// "control" section.
+func GRPC(c *config.Config) GRPCConfig {
+	return GRPCConfig{
+		c.Sub(subsection).Sub(grpcSubsection),
+	}
+}
+
+// Endpoint returns value of "endpoint" config parameter
```

Comment:
We don't need to specify parent sections here:
```suggestion
// Endpoint returns value of "endpoint" config parameter.
```

Reply:
Fixed.